packages/circuits/src/ReportIdentityProof.ts

```diff
@@ -0,0 +1,22 @@
+import { BaseProof, Prover } from '@unirep/circuits'
+import { Groth16Proof } from 'snarkjs'
+
+export class ReportIdentityProof extends BaseProof {
+    public reportNullifier: bigint
+    public attesterId: bigint
+    public epoch: bigint
+    public stateTreeRoot: bigint
+
+    constructor(
+        publicSignals: (bigint | string)[],
+        proof: Groth16Proof,
+        prover?: Prover
+    ) {
+        super(publicSignals, proof, prover)
+        this.reportNullifier = this.publicSignals[0]
```

Comment:
we can add a `idx` to for configuring the publidSignal idx to improve the readability here.

just like what they have in epochKeyProof:
https://github.com/Unirep/Unirep/blob/e57834c1244e76a1e3e8289b0f92b072c2fc8fb8/packages/circuits/src/EpochKeyProof.ts#L13-L18

Reply:
fixed